_includes/release-notes/v3.15.0-release-notes.md

```diff
@@ -0,0 +1,46 @@
+19 Jun 2020
+
+#### Encryption using Wireguard is generally available
+
+Calico v3.14 introduced tech-preview support for encryption using Wireguard. In v3.15, we’re excited to announce that this feature is promoted to GA.
+To give it a spin, head on over to [the encryption how-to guide](https://docs.projectcalico.org/master/security/encrypt-cluster-pod-traffic).
```

Comment:
Looks like the link here is master rather than v3.15

Reply:
Please do update in your gdoc too, assuming that's the right thing.